src/libstd/os/linux/raw.rs

```diff
@@ -155,6 +155,11 @@ mod arch {
     }
 }
 
+#[cfg(target_arch = "mips64")]
+mod arch {
+    pub use libc::{off_t, ino_t, nlink_t, blksize_t, blkcnt_t, stat, time_t};
```

Comment:
It's actually ok to just leave this out, the deprecated modules are just here for backwards compatibility. MIPS64 didn't even work back then, so there's no compatibility to be had!